ios/Classes/BetterPlayer.m

```diff
@@ -249,6 +249,13 @@ - (void)setDataSourcePlayerItem:(AVPlayerItem*)item withKey:(NSString*)key{
     _stalledCount = 0;
     _isStalledCheckStarted = false;
     _playerRate = 1;
+    
+    // Prevent pip change to black screen when play video from notification center
+    // When add video output to player item, maybe it makes player item high priority
+    // and player still generates frame when not visible
+    AVPlayerItemVideoOutput* playerItemVideoOutput = [[AVPlayerItemVideoOutput alloc] initWithPixelBufferAttributes:@{(id)kCVPixelBufferPixelFormatTypeKey: @(kCVPixelFormatType_32ARGB)}];
```

Comment:
Sorry.
How did you know the code?
If you have any reference, please add the link to PR description 🙏 

And I am afraid,
Now we don't have any "close" process?

- We call `addOutput`, so we don't need `removeOutput`?
- `playerItemVideoOutput` is "alloc"ed and "init"ed. We don't need to dispose it? (I'm afraid of memory leaks.)

Reply:
@noboru-i 

> If you have any reference, please add the link to PR description

I added it 🙏

> Now we don't have any "close" process?
> We call addOutput, so we don't need removeOutput?
> playerItemVideoOutput is "alloc"ed and "init"ed. We don't need to dispose it? (I'm afraid of memory leaks.)

Thank you for pointing out 👍

I checked again. In this case, only add `video output` for `player item` (not add any observers or other handling...) so it should be auto-released by IOS memory management system ([ARC](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/MemoryMgmt.html))
- `Player Item` is the only obj that contains `AVPlayerVideoOuput` (reference count = 1), so when `Player Item` is released -> `AVPlayerVideoOuput` should be released too.

For make sure, I use tool `Debug Memory Graph` on XCode and checked this case was not leaked
- Play, change data source -> only 1 obj in-memory tree
- Back to list screen -> can not found obj (obj is released)


<img width="1425" alt="Screen Shot 2023-07-10 at 15 15 59" src="https://github.com/dwango-nfc/betterplayer/assets/38061170/2057cee0-66de-42bc-92be-9e1d6565547d">

Reply:
Thank you for adding and checking it detail.
I understand 👍